code/src/components/Form.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+
+import FirstQuestion from './FirstQuestion'
+import SecondQuestion from './SecondQuestion'
+import ThirdQuestion from './ThirdQuestion'
+import FourthQuestion from './FourthQuestion'
+import FifthQuestion from './FifthQuestion'
+import SixthQuestion from './SixthQuestion'
+import SeventhQuestion from './SeventhQuestion'
+import TheEnd from './TheEnd'
+import FirstPage from './FirstPage'
+
+import 'index.css';
+import YourProfessional from './YourProfessional';
+
+const Form = () => {
+  //  const states
+  const [counter, setCounter] = useState(0);
+  const [nameInput, setNameInput] = useState('');
+  const [brushInput, setBrushInput] = useState();
+  const [frequencyInput, setFrequencyInput] = useState();
+  const [checkboxGroup, setCheckboxGroup] = useState([]);
+  const [goalInput, setGoalInput] = useState();
+  const [flossInput, setFlossInput] = useState();
+  const [requestInput, setRequestInput] = useState();
+
+  //  functions
+  const onNextQuestionChange = () => {
+    setCounter(counter + 1)
+  }
+
+  const onPreviousQuestionChange = () => {
+    setCounter(counter - 1)
+  }
+
+  const onNameInputChange = (event) => {
+    setNameInput(event.target.value)
+  }
+
+  const onBrushInputChange = (event) => {
+    setBrushInput(event.target.value)
+  }
+
+  const onFrequencyInputChange = (event) => {
+    setFrequencyInput(event.target.value)
+  }
+
+  const onCheckboxGroupToggle = (checkboxValue) => {
+    if (checkboxGroup.includes(checkboxValue)) {
+      setCheckboxGroup(checkboxGroup.filter((item) => item !== checkboxValue))
+    } else {
+      setCheckboxGroup([checkboxValue, ...checkboxGroup])
+    }
+  }
+
+  const onFlossInputChange = (event) => {
+    setFlossInput(event.target.value)
+  }
+
+  const onGoalInputChange = (event) => {
+    setGoalInput(event.target.value)
+  }
+
+  const onRequestInputChange = (event) => {
+    setRequestInput(event.target.value)
+  }
+
+  // return question carousel
+  return (
+    <main className="main-contaniner">
+      {counter === 0 && <FirstPage onNextQuestionChange={onNextQuestionChange} />}
```

Comment:
<FirstPage could be moved one step down to make all counters === look the same.

Reply:
Thank you Kristin for the feedback ⭐️
You are right about this, that would improve the readability a bit. I will look into this. Thank you for pointing it out and going through my code! :)